src/lib/rest/rest.cpp

```diff
@@ -871,6 +871,11 @@ static std::string apiVersionGet(const char* path)
     return "v2";
   }
 
+  if (strcmp(path, "/admin/log") == 0)
+  {
+    return "v2";
```

Comment:
In that case, a comment is neded in order to explain the need of fooling ourseleves returning "v2" in this case.